metadata-transformer/src/test/java/no/unit/nva/metadata/service/MetadataServiceTest.java

```diff
@@ -181,24 +183,35 @@ void getCreatePublicationReturnsNoValueWhenDcTermsElementIsUnknown() throws IOEx
         assertTrue(request.isEmpty());
     }
 
-    @ParameterizedTest(name = "Identifiers that kind {0} are processed as DOIs")
+    @ParameterizedTest(name = "Identifiers with property {0} that kind {1} are processed as DOIs")
```

Comment:
that kind?

Reply:
Will fix, and add link to DOI-display documentation in comment